### ISSUE TITLE
Mouse cursor disappearing 

### ISSUE THREAD

**Issue by ZenoMetal** [comp500/BorderlessMining#9]
As the Title says, my cursor vanishes when I enable fullscreen

**Comment by comp500** [comment]
What operating system and hardware are you using?

**Comment by ZenoMetal** [comment]
Windows 10, what Hardware is important to know for you? I use a GTX 1070 and an Intel i7-8700 and I play on 1080p on a 1080p screen, I have no scaling enabled in Windows or something I know of that coud cause this

**Comment by comp500** [comment]
Hmm, that's odd... Does the same happen in fullscreen without the mod? Also try going in game, and opening/closing the menu.

**Comment by ZenoMetal** [comment]
It only happens while using the mod, without it in normal fullscreen everything is fine. Restarting or joining worlds/servers also didn't change anything. I can still click stuff, the cursor just goes invisible

**Comment by comp500** [comment]
Do you have any overlays running? Overwolf/Discord/Windows Game Bar/Steam/NVIDIA share etc.. try disabling those

**Comment by ZenoMetal** [comment]
Nope, none are enabled and I'm still having the issue

**Comment by tdpeuter** [comment]
Yes, I also experience this problem since updating to 1.16.4 (and still in 1.16.5) on Windows around Christmas. 
Occurence: Cursor shows inconsistently when playing fullscreen. Most of the time (when starting in fullscreen) it is visable, but after a while dissapears, I can still click everything, but gets confusing because I can't see where I'm clicking other than highlights that happen such as buttons. I am yet trying to figure out if this occurs when returning after switching screens (I use alt + tab), when I join new world or just after time... 
I normally play on fullscreen, but find myself playing on maximized window instead since this started happening. This of course takes away the entire point of the mod. Other than several mods, I use f.lux. I do not use any Discord overlays/other overlays from third party apps (media player overlays don't show in-game). Some mods of mine have overlays or visual elements, but this didn't cause any issues in previous versions: [Carpet mod](https://github.com/gnembon/fabric-carpet), [Falling Leaves](https://www.curseforge.com/minecraft/mc-mods/falling-leaves-fabric), [Illuminations](https://www.curseforge.com/minecraft/mc-mods/illuminations), [Minihud](https://www.curseforge.com/minecraft/mc-mods/minihud/), [Tweakeroo Mod](https://www.curseforge.com/minecraft/mc-mods/tweakeroo). I also run [OptiFabric](https://www.curseforge.com/minecraft/mc-mods/optifabric/). 
I updated my mouse's drivers and iCUE software is also up to date. I also tried something with a cursor trail, but that didn't change anything. I run a i7 9600K with a RTX2060 and 16GB RAM, 12GB enabled to Java.
@comp500 When did this start in your case? Are you running any mods? Does this always happen as soon as you start the game or is it occasional?
@ZenoMetal Are there any known incompatibilities with your mod and others? I also presume, based on your reactions, that this is an unknown issue.

**Comment by NebelNidas** [comment]
By any chance, do you have OBS running in the background? I've also experienced my cursor disappearing with OBS open, although that also happened in vanilla.

**Comment by tdpeuter** [comment]
Yes, that's an issue indeed, but this can happen even without OBS running. It would even disappear while the Minecraft window is active, so it turns out this is not related to switching windows. 
Another update: OptiFabric doesn't cause the issue, because I do no longer is it and the problem still occurs.

**Comment by znopp** [comment]
Having the same issue as @ZenoMetal - Enabling the mod and pressing f11 to go into windowed borderless mode, my cursor disappears. But, if I move it over to my second monitor, I can see it again, almost as if the game is rendering itself above the cursor. Happens with my 1.19.2, 1.19 and 1.18 instances, the latter two have not been updated in a very long time, meaning it is not a mod incompatibility nor version issue. Double checked and used the thanos method to see, and fabric api with borderless mining still caused the issue. Tried 1.1.3, 1.1.4 and 1.1.5 to no avail. To me it feels like a windows 10 issue, but what do I know.

**Comment by Dight67** [comment]
If anyone will look here in future and happen to use f.lux, disable "Software mouse cursor when needed" in options, this fixed that issue for me.

**Comment by AppleJuice80** [comment]
Yeah it's happening to me as well :(
